pkg/server/test/read_changes.go

```diff
@@ -0,0 +1,74 @@
+package test
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	"github.com/oklog/ulid/v2"
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+
+	"github.com/openfga/openfga/pkg/server/commands"
+	"github.com/openfga/openfga/pkg/storage"
+	"github.com/openfga/openfga/pkg/testutils"
+	"github.com/openfga/openfga/pkg/tuple"
+)
+
+func BenchmarkReadChanges(b *testing.B, ds storage.OpenFGADatastore) {
+	inputModel := `
```

Comment:
Could you put `b.StopTimer()` before all the setup steps, and `b.StartTimer()` right before `b.Run()`? 

https://pkg.go.dev/testing#B.StopTimer
https://pkg.go.dev/testing#B.StartTimer